modules/microshift-config-yaml.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * microshift/using-config-tools.adoc
+
+:_content-type: CONCEPT
+[id="microshift-config-yaml_{context}"]
+= Using a YAML configuration file
+
+Configure your {product-title} using a YAML file. The values specified in your configuration file are used when you execute a command.
+// Q: can these values be overriden with CLI flags?
+
+[id="microshift-yaml-default_{context}"]
+== Default settings
+If you do not create a `config.yaml` file, the default values are used.
+// Q: what creates this default yaml?
+// Q: can default file be modified? (instead of making a new one)
+
+.Default YAML file example
+[source,yaml]
+----
+cluster:
+  clusterCIDR: 10.42.0.0/16
+  serviceCIDR: 10.43.0.0/16
+  serviceNodePortRange: 30000-32767
+  dns: 10.43.0.10
+  domain: cluster.local
+  url: https://127.0.0.1:6443
+nodeIP: ""
+nodeName: ""
+logVLevel: 0
+----
+
+[IMPORTANT]
+====
+The configuration file must be located at the user-specific directory, `~/.microshift/config.yaml`, and the system-wide `/etc/microshift/config.yaml` directory. An existing user-specific `config.yaml` takes precedence.
+====
+//Q: can this be done with a command, or is this an instance where user goes into the directory and creates a text file?
+
+[id="microshift-yaml-format-example_{context}"]
+== Formatting a customized YAML file
+
+.Procedure
+
+* Create a YAML file with the following formatting:
```

Comment:
I missed this before, should have a `.` since there are two steps (so far)
```suggestion
. Create a YAML file with the following formatting:
```

Reply:
Nah, that was me. I added one. Really trying hard to _not_ keep picking it for the sake of getting the structure merged, but sometimes I can't resist.